pkg/console/operator/status.go

```diff
@@ -107,33 +151,6 @@ func (c *consoleOperator) logConditions(conditions []operatorsv1.OperatorConditi
 	}
 }
 
-// setStatusCondition
-// A generic helper for setting a status condition.
-// To use when another more specific status function is not sufficient.
-// examples:
-//  setStatusCondition(operatorConfig, Failing, True, "SyncLoopError", "Sync loop failed to complete successfully")
-func (c *consoleOperator) SetStatusCondition(operatorConfig *operatorsv1.Console, conditionType string, conditionStatus operatorsv1.ConditionStatus, conditionReason string, conditionMessage string) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               conditionType,
-		Status:             conditionStatus,
-		Reason:             conditionReason,
-		Message:            conditionMessage,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-func (c *consoleOperator) ConditionResourceSyncSuccess(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeDegraded,
-		Status:             operatorsv1.ConditionFalse,
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
 func (c *consoleOperator) ConditionDeploymentAvailable(operatorConfig *operatorsv1.Console, message string) *operatorsv1.Console {
```

Comment:
Wont this change also enable to remove all of these helpers ?

Reply:
I would like to remove them all, agree.  Without any structural change anywhere else, so 1:1 replacement.

Reply:
I may do this in several PRs, this one can concentrate on updating `Degraded`.  That way we can move along with others like #253